custom_components/bambu_lab/translations/en.json

```diff
@@ -360,6 +360,11 @@
           }
         }
       }
+    },
+    "camera": {
+      "camera": {
+        "name": "Camera"
```

Comment:
If we don't add this to the fr-fr file, does it just use the English one instead? I.e. Do we have to add something to all the non-English json files or can localization come later (from native speakers)?